configs/BETAFPVF405/config.h

```diff
@@ -39,6 +39,11 @@
 #define USE_FLASH
 #define USE_FLASH_W25Q128FV
 #define USE_MAX7456
+#ifndef USE_MAG
+#define USE_MAG
+#define USE_MAG_QMC5883
+#define USE_MAG_IST8310
+#endif
```

Comment:
If this board is not part of a BNF including MAG, this should be a cloud build option.
```suggestion
```